packages/react-ui/src/icons/components/bicolor/Agreement.js

```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import * as React from "react";
```

Comment:
what are all these changes ?

Reply:
Probably just a bump in the package which generates these automatically